Umbraco-Cloud/Set-Up/Media/index.md

```diff
@@ -127,7 +128,9 @@ Update `~/config/imageprocessor/cache.config` by removing the default “DiskCac
           <settings>
             <setting key="CachedStorageAccount" value="DefaultEndpointsProtocol=https;AccountName=;AccountKey=" />
             <setting key="CachedBlobContainer" value="cache" />
-            <setting key="CachedCDNRoot" value="http://<your Azure CDN>.vo.msecnd.net/" />
+            <setting key="CachedCDNRoot" value="https://<your Azure CDN>.vo.msecnd.net/" />
+            <setting key="CachedCDNTimeout" value="2000" />
+            <setting key="UseCachedContainerInUrl" value="false" />
```

Comment:
I think UseCachedContainerInUrl should be "true" by default. Only if you have used a "Origin path" when setting up the CDN endpoint would you want to set it to "false"?

Reply:
@Jeavon Makes sense - I mentioned this in the issue as well:

"Also set `UseCachedContainerInUrl` to false by default, as the latest ImageProcessing docs suggest newer Azure formats require that. Though we're using a couple of implementation currently with that set to true without issue, so you're probably in the best place to confirm it either way."

I'll update and push up shortly :)

Reply:
I've just provisioned a new CDN to double check too

Reply:
From what I can see in our instances, it should be the recommendation to be `true` by default, unless your tests suggest otherwise?